CONTRIBUTING.rst

```diff
@@ -62,29 +75,29 @@ Ready to contribute? Here's how to set up `renku` for local development.
 
       $ git clone git@github.com:your_name_here/renku.git
 
-3. Install your local copy into a virtualenv. Assuming you have
-   virtualenvwrapper installed, this is how you set up your fork for local
-   development:
+3. Ensure you have your development environment set up. For this we encourage usage of `pipenv`
+(other virtual environment tools could be used as well) `pyenv`:
```

Comment:
`pyenv` is kind of coming out of the blue for me here - aren't you talking about pipenv?